Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	
+++ b/.gitignore	
@@ -1,0 +1,9 @@
+target/
+*.class
+*.jar
+*.war
+*.ear
+*.logs
+*.iml
+.idea/
+.eclipse
\ No newline at end of file
Index: core_java/grep/log/grep.out
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>    Is father, mother, Tybalt, Romeo, Juliet,\nEnter Romeo and Juliet aloft, at the Window.\n    And Romeo dead; and Juliet, dead before,\n    Romeo, there dead, was husband to that Juliet;\nRomeo fait des clins d'oeil à Juliette\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/core_java/grep/log/grep.out b/core_java/grep/log/grep.out
--- a/core_java/grep/log/grep.out	
+++ b/core_java/grep/log/grep.out	
@@ -3,3 +3,8 @@
     And Romeo dead; and Juliet, dead before,
     Romeo, there dead, was husband to that Juliet;
 Romeo fait des clins d'oeil à Juliette
+    Is father, mother, Tybalt, Romeo, Juliet,
+Enter Romeo and Juliet aloft, at the Window.
+    And Romeo dead; and Juliet, dead before,
+    Romeo, there dead, was husband to that Juliet;
+Romeo fait des clins d'oeil à Juliette
Index: core_java/jdbc/pom.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/core_java/jdbc/pom.xml b/core_java/jdbc/pom.xml
--- a/core_java/jdbc/pom.xml	
+++ b/core_java/jdbc/pom.xml	
@@ -1,0 +1,20 @@
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>ca.jrvs.apps</groupId>
+    <artifactId>jdbc</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <java.version>11</java.version>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>9.1-901-1.jdbc4</version>
+        </dependency>
+    </dependencies>
+</project>
\ No newline at end of file
Index: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JDBCExecutor.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package ca.jrvs.apps.jdbc;public class JDBCExecutor {\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JDBCExecutor.java b/core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JDBCExecutor.java
--- a/core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JDBCExecutor.java	
+++ b/core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JDBCExecutor.java	
@@ -1,2 +1,8 @@
-package ca.jrvs.apps.jdbc;public class JDBCExecutor {
+package ca.jrvs.apps.jdbc;
+
+public class JDBCExecutor {
+
+    public static void main(String[] args) {
+        System.out.println("Hello Learning JDBC");
+    }
 }
